runtime/cere/src/lib.rs

```diff
@@ -1410,9 +1411,11 @@ pub type CheckedExtrinsic = generic::CheckedExtrinsic<AccountId, RuntimeCall, Si
 
 /// Runtime migrations
 type Migrations = (
+	pallet_ddc_clusters::migrations::v3::MigrateToV3<Runtime>,
 	pallet_ddc_clusters::migrations::v2::MigrateToV2<Runtime>,
 	pallet_ddc_staking::migrations::v1::MigrateToV1<Runtime>,
 	pallet_ddc_customers::migration::MigrateToV2<Runtime>,
+	pallet_ddc_customers::migration::MigrateToV1<Runtime>,
```

Comment:
`pallet_ddc_customers::migration::MigrateToV1` has been executed on Mainnet. You should remove it from the list.